### DISCUSSION TITLE
wrong data issue when using libre + calibration plugin + non linear calibration

### DISCUSSION THREAD

**Discussion by zwidow** [NightscoutFoundation/xDrip#3827] [Bug reports]
happens in all versions

When using libre + calibration plugin + non linear calibration the glucose level used to calculate all data are wrong. 
For alla data i mean: all stats and all data sended to cloud.

Instead using the plug-in value it uses the xdrip original and this cause wrong average calculation and wrong HbA1c % etc.

**Comment by PhilTU** [comment]
A better title might be "xDrip+ ignores calibration in data upload"

**Reply by zwidow** [reply]
calibrations are used and sent, but it use the linear calibration always (default xdrip) instead the non linear, when is selected/used.
